frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -209,10 +210,13 @@ export const Chart = ({
   };
 
   const removeChart = () => {
-    if (chart.current) {
-      chart.current.remove();
-      chart.current = undefined;
+    if (chartInitialized.current) {
       window.removeEventListener('resize', onResize);
+      chart.current?.timeScale().unsubscribeVisibleLogicalRangeChange(calculateChange);
+      chart.current?.unsubscribeCrosshairMove(handleCrosshair);
```

Comment:
imo. `chart.remove()` should clean up everything but from a quick look at the function documentation I don't know if it does so I left the unsubscribing in here just for safety..wdyt @thisconnect should I remove these?